### ISSUE TITLE
Explore and document how to set up SubQuery

### ISSUE THREAD

**Issue by kapetan3sid** [polkadot-developers/polkadot-docs#276]
Create a step-by-step guide on how to set up a project with the rest of the configuration so that developers can index data from testnet

**Comment by kianenigma** [comment]
Is this related to contracts? sorry, not super familiar with SubQuery.

**Comment by kapetan3sid** [comment]
Blocked by [Infinite loop when querying non-existing Runtime API with the Reconnecting Client
closed
 ](https://github.com/paritytech/subxt/issues/1903)

**Comment by niklasad1** [comment]
Should be fixed if you update subxt v0.38.1

**Comment by kapetan3sid** [comment]
there is another issue pending @niklasad1 https://github.com/paritytech/contract-issues/issues/29
